Week 3

BentonFraizer opened the pull request.

BentonFraizer committed:
Создано полноэкранное меню

Создано полноэкранное меню для разрешений планшетов и телефонов.
Также устранено замечание касающееся :hover трубки телефона в header.
Теперь при наведении указателя мыши трубка перекрашивается в черный
цвет.

BentonFraizer committed:
Промежуточный результат на проверку (.svg)

Почему-то не работает часть svg-картинок.
1. Иконка батончика с элемента слайдера у которого есть ховер с составом
батончика;
2. Иконка бобов (последняя срока в ховере элемента слайдера);
3. И если не ошибаюсь, то стрелочки переключения слайдера тоже не
получается добавить как svg.
На данном этапе выполнена вёрстка секции со слайдером, но слайдер пока
не реализован.

BentonFraizer committed:
Завершено выполнение заданий недели №3

В этой части закончена реализация работы не цикличного слайдера,
свойство transition для плавного перелистывания слайдов работает.

b-ars07 commented:
1. скролл должен быть отключен, при открытом меню - 
![image](https://user-images.githubusercontent.com/9081116/87099092-efd7da00-c259-11ea-9f34-1d6fd1cd2703.png)

BentonFraizer committed:
Устранено замечание по поводу скролла.

Выполнен функционал согласно которого при открытии полноэкранного меню
пользователю запрещается пользоваться скролом страницы.
Заменил способ перелистывания слайдера, поскольку попиксельный вариант
нам не подходит. При изменении размеров окна, слайдер работал
некорректно.
Также переделан адаптив для секции со слайдером,т.к. после правки
html-разметки сбились стили.
